solution/zenko-cr.yaml

```diff
@@ -0,0 +1,48 @@
+---
```

Comment:
This is not a Zenko CR but a Zenko Version CR

Reply:
oh, the contents are right though just the name is wrong?

Reply:
The content's fine, it's just that the name is misleading regarding what it is.

There is the question of whether we consider this file to be the single source of truth regarding image versions contained in a global Zenko release, or if we want this to be generated from a more canonical source. Both options are open.

Reply:
I have renamed it, for the time being, it is used as a partial source containing some of the dependencies where the others are placed in dep.txt. Kind of a weird split but since this file was in another repo I thought it was better to use it as much as I could and create another file for what is missing. A single source of truth for all dependencies would be nice though.